packages/utils/src/groupTransactionsByDay.ts

```diff
@@ -2,7 +2,7 @@ import _ from 'lodash';
 import moment from 'moment';
 import { Statement } from '@deriv/api-types';
```

Comment:
Shouldn't these packages be listed as a dependency of this package?

Reply:
And, for lodash, Isn't it better to just install `groupby` package instead of `lodash` itself?